src/pages/Taiyo/main.ts

```diff
@@ -0,0 +1,120 @@
+import { pageInterface } from '../pageInterface';
+
+const uiSelec = '.flex.justify-end.gap-2';
+
+const { asyncWaitUntilTrue: awaitUi, reset: resetAwaitUi } = utils.getAsyncWaitUntilTrue(
+  () => j.$(uiSelec).length,
+);
+
+const { asyncWaitUntilTrue: awaitReader, reset: resetawaitReader } = utils.getAsyncWaitUntilTrue(
+  () => j.$('.grid-areas-mediaChapter img').length,
+);
+
+export const Taiyo: pageInterface = {
+  name: 'Taiyo',
+  domain: 'https://taiyo.moe',
+  languages: ['Portuguese'],
+  type: 'manga',
+  isSyncPage(url) {
+    if (url.split('/')[3] === 'chapter') {
+      return true;
+    }
+    return false;
+  },
+  isOverviewPage(url) {
+    if (url.split('/')[3] === 'media') {
+      return true;
+    }
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('.media-title').text();
+    },
+    getIdentifier(url) {
+      return url.split('/')[4];
```

Comment:
the identifier has to be the same as on the overview

Reply:
you could do return Taiyo.overview!.getIdentifier(Taiyo.sync.getOverviewUrl(url));